cancermuts/datasources.py

```diff
@@ -1467,53 +1463,50 @@ def _get_mobidb_disorder_predictions_assignments(self, sequence, *args, **kwargs
         data = self._get_mobidb(sequence, use_alias=use_alias)
 
         try:
-            disorder_data = data['mobidb_consensus']['disorder']
+            curated_disorder_data = data['curated-disorder-priority']
         except KeyError:
-            self.log.error("No disorder data was found")
+            self.log.error("No curated disorder data was found.")
+            return None
+        
+        try:
+            predicted_disorder_data = data['prediction-disorder-priority']
+        except KeyError:
+            self.log.error("No predicted disorder data was found.")
             return None
```

Comment:
In line 1479 and 1491 we need to check if there is data available, which is currently done with `if .. is not None`. Is there another way?

Reply:
Wait, I might have confused myself

Reply:
yes, my point is different - here we have:

```
        try:
            curated_disorder_data = data['curated-disorder-priority']
        except KeyError:
            self.log.error("No curated disorder data was found.")
            return None
        
        try:
            predicted_disorder_data = data['prediction-disorder-priority']
        except KeyError:
            self.log.error("No predicted disorder data was found.")
            return None
```

this section `return`s if either data from predictions or curations isn't found; however, we want to have *at least* one of them to be present rather than necessarily both, since we want e.g. the prediction information even though the curated data isn't available